packages/cli/src/index.js

```diff
@@ -8,9 +8,10 @@
  */
 
 // gracefulify() has to be called before anything else runs
-require('graceful-fs').gracefulify(require('fs'));
+// eslint-disable-next-line no-unused-vars
+import gracefulFs from './util/gracefulFs';
```

Comment:
I don't see a reason to extract this to a util, and in this form you import with side-effects. Please inline the implementation here

Reply:
@thymikee the problem here is the 
```
import cli from './cliEntry';
```
cause we need to run `gracefulFs.gracefulify(realFs);` before it

Reply:
@thymikee keep extracted?